MobileWalletTests/TariLibWrapperTests.swift

```diff
@@ -432,7 +433,8 @@ class TariLibWrapperTests: XCTestCase {
                 databaseName: dbName,
                 publicAddress: address,
                 discoveryTimeoutSec: TariSettings.shared.discoveryTimeoutSec,
-                safMessageDurationSec: TariSettings.shared.safMessageDurationSec
+                safMessageDurationSec: TariSettings.shared.safMessageDurationSec,
+                networkName: "weatherwax"
```

Comment:
We keep network names in `TariNetwork` enum. Please consider this suggestion:
```suggestion
               networkName: TariNetwork.weatherwax.rawValue
```